src/main/resources/application.yml

```
@@ -40,4 +40,5 @@ jwt:
   refresh-token-validity: 604800000
 
 domain:
-  base-url: ${DOMAIN_BASE_URL}
\ No newline at end of file
+  base-url: ${DOMAIN_BASE_URL}
+  dev-base-url: ${DEV_DOMAIN_BASE_URL}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)